pkg/model/resource/options_test.go

```diff
@@ -73,24 +74,37 @@ var _ = Describe("Resource Options", func() {
 			Expect(options.Validate().Error()).To(ContainSubstring("kind cannot be empty"))
 		})
 
-		It("should fail if the Kind is not pascal cased", func() {
-			// Base case
-			options := &Options{Group: "crew", Version: "v1", Kind: "FirstMate"}
-			Expect(options.Validate()).To(Succeed())
-
-			// Can't detect this case :(
-			options = &Options{Group: "crew", Version: "v1", Kind: "Firstmate"}
-			Expect(options.Validate()).To(Succeed())
+		DescribeTable("valid Kind values - according to core Kubernetes",
+			func(kind string) {
+				options := &Options{Group: "crew", Kind: kind, Version: "v1"}
+				Expect(options.Validate()).To(Succeed())
+			},
+			Entry("camelcase", "FirstMate"),
+			Entry("more than one caps at the start", "FIRSTMate"),
+		)
```

Comment:
IMHO, I would not use `DescribeTable` here as all the rest of the tests are using raw `Describe`, `Context` and `It`. The added benefit of using `DescribeTable` also seems pretty low, as it is a 2LoC function and 2 entries. So I would just stick to what it is being used already. But thats just an opinion.